src/pages/_app.tsx

```diff
@@ -1,8 +1,10 @@
+import "@styles/globals.css";
 import type { AppProps } from "next/app";
 import { ThemeProvider } from "next-themes";
 import Head from "next/head";
 import { useEffect, useState } from "react";
-// import "@fontsource/karla";
+import "@fontsource/sora";
+import Layout from "@components/Layout/Layout";
 
 export default function MyApp({ Component, pageProps }: AppProps) {
   const [showChild, setShowChild] = useState(false);
```

Comment:
The state showChild is used to conditionally render the Component and Layout only when the useEffect hook has been completed. This is because useEffect runs after the initial render of the component, and by setting setShowChild(true) within it, we are indicating that the element is ready to render.

This is a common technique used in Next.js to ensure that any server-side code that needs to be executed has been completed before rendering the client-side components.

Using this technique, we avoid potential client-side errors that could occur if the component tries to access data or perform actions before they have been fetched or loaded.

Reply:
Okay, makes sense.